compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__add_missing_patterns_bool.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "\npub fn main() {\n  let bool = True\n  case bool {}\n}\n"
+---
+----- BEFORE ACTION
+
+pub fn main() {
+  let bool = True
+  case bool {}
+  ▔▔▔▔▔↑      
+}
+
+
+----- AFTER ACTION
+
+pub fn main() {
+  let bool = True
+  case bool {
+    _ -> todo
```

Comment:
This isn't very helpful. I would be very frustrated if the action gave me this.

Is there a way to add the constructors here?

Reply:
Unfortunately this is more a problem with the exhaustiveness checker. For this code:
```gleam
pub fn main() {
  case True {
    
  }
}
```

The compiler produces this error:
```
error: Inexhaustive patterns
  ┌─ /home/gears/projects/temp/testing_gleam/src/testing_gleam.gleam:5:3
  │  
5 │ ╭   case True {
6 │ │     
7 │ │   }
  │ ╰───^

This case expression does not have a pattern for all possible values. If it
is run on one of the values without a pattern then it will crash.

The missing patterns are:

    _
```

I was planning to open another issue after this about improving these suggestions, but should I open that now and leave this on hold until that is resolved?

Reply:
Aye, I understand what the codebase does today. What I mean is that to be a useful addition this code action needs to be able to add constructors for empty case expressions.